servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -170,14 +175,11 @@ def _batch_collect_components(self, instance_info, offset):
 
         :return: dict, raw response from CMDB
         """
-
         auth = (instance_info.user, instance_info.password)
         url = instance_info.url + '/api/now/table/cmdb_ci'
-        sys_class_filter_query = self.get_sys_class_component_filter_query(instance_info.include_resource_types)
-        if sys_class_filter_query:
-            params = {'sysparm_query': sys_class_filter_query}
-        else:
-            params = {}
+        sys_class_filter_query = self._get_sys_class_component_filter_query(instance_info.include_resource_types)
+        params = self._params_append_to_sysparm_query(add_to_query=sys_class_filter_query)
+        params = self._params_append_to_sysparm_query(add_to_query=instance_info.cmdb_ci_sysparm_query, params=params)
```

Comment:
I miss a test when batch collecting components which ensure that the `sysparm` is set correctly if we configure the query filtering

Reply:
added.